controller.py

```diff
@@ -361,8 +361,7 @@ def _areas(self, areas):
         return result
 
     def _format_place_name(self, place):
-        parent_name = (place.parent.abbreviated_name or place.parent.external_name) if place.parent else "unknown"
-        return "%s (%s)" %(place.external_name, parent_name)
+        return place.human_friendly_name or 'Everywhere'
```

Comment:
A library like Internet Archive whose service area is 'everywhere' has no human-friendly name. This is fine for display to prospective patrons, but I figured we should be more explicit in the admin interface.